frontend/packages/legacy-css/font.scss

```diff
@@ -0,0 +1,145 @@
+'Noto Sans Kannada', 'Noto Sans Khmer', 'Noto Sans KR', 'Noto Sans Myanmar', 'Noto Sans Sinhala',
+'Noto Sans Tamil', 'Noto Sans Telugu', 'Noto Sans Thai', 'Noto Sans Thaana';
+
+$main-font: $metropolis-font, $noto-sans-fonts, sans-serif;
+
+$thin-font-weight: 100;
+$extra-light-font-weight: 200;
+$light-font-weight: 300;
+$regular-font-weight: 400;
+$medium-font-weight: 500;
+$semi-bold-font-weight: 600;
+$bold-font-weight: 700;
+$extra-bold-font-weight: 800;
+$black-font-weight: 900;
+
+@mixin main-font-thin {
+  font-family: $main-font;
+  font-weight: $thin-font-weight;
+  font-style: normal;
+}
+
+@mixin main-font-thin-italic {
+  @include main-font-thin;
+  font-style: italic;
+}
+
+@mixin main-font-extra-light {
+  font-family: $main-font;
+  font-weight: $extra-light-font-weight;
+  font-style: normal;
+}
+
+@mixin main-font-extra-light-italic {
+  @include main-font-extra-light;
+  font-style: italic;
+}
+
+@mixin main-font-light {
+  font-family: $main-font;
+  font-weight: $light-font-weight;
+  font-style: normal;
+}
+
+@mixin main-font-light-italic {
+  @include main-font-light;
+  font-style: italic;
+}
+
+@mixin main-font-regular {
+  font-family: $main-font;
+  font-weight: $regular-font-weight;
+  font-style: normal;
+}
+
+@mixin main-font-regular-italic {
+  @include main-font-regular;
+  font-style: italic;
+}
+
+@mixin main-font-medium {
+  font-family: $main-font;
+  font-weight: $medium-font-weight;
+  font-style: normal;
+}
+
+@mixin main-font-medium-italic {
+  @include main-font-medium;
+  font-style: italic;
+}
+
+@mixin main-font-semi-bold {
+  font-family: $main-font;
+  font-weight: $semi-bold-font-weight;
+  font-style: normal;
+}
+
+@mixin main-font-semi-bold-italic {
+  @include main-font-semi-bold;
+  font-style: italic;
+}
+
+@mixin main-font-bold {
+  font-family: $main-font;
+  font-weight: $bold-font-weight;
+  font-style: normal;
+}
+
+@mixin main-font-bold-italic {
+  @include main-font-bold;
+  font-style: italic;
+}
+
+@mixin main-font-extra-bold {
+  font-family: $main-font;
+  font-weight: $extra-bold-font-weight;
+  font-style: normal;
+}
+
+@mixin main-font-extra-bold-italic {
+  @include main-font-extra-bold;
+  font-style: italic;
+}
+
+@mixin main-font-black {
+  font-family: $main-font;
+  font-weight: $black-font-weight;
+  font-style: normal;
+}
+
+@mixin main-font-black-italic {
+  @include main-font-black;
+  font-style: italic;
+}
+
+$barlowSemiCondensed-semibold: 'Barlow Semi Condensed Semibold', $noto-sans-fonts, sans-serif;
+$barlowSemiCondensed-medium: 'Barlow Semi Condensed Medium', $noto-sans-fonts, sans-serif;
+
+$code-font: 'Source Code Pro', Monaco, 'Bitstream Vera Sans Mono', 'Lucida Console', Terminal, monospace;
+
+/*
+   Font Awesome S3 Bucket URLs
+   Current version: 6.6.0
+   If you are changing the location of these files,
+   see the readme in this directory for more detail on manual steps to take when uploading new files.
```

Comment:
Copy over this readme from `shared/css` (or remove this comment)?

Reply:
@levadadenys What do you think?